tests/common/helpers/enable-dynamic-buffer.py

```diff
@@ -119,6 +120,10 @@ def stop_traditional_buffer_model(config_db):
     # Stop the buffermgrd
     # We don't stop the buffermgrd at the beginning
     # because we need it to remove tables from APPL_DB while their counter part are removed from CONFIG_DB
+
+    # Before stopping buffermgrd, need to make sure buffermgrd is running,
+    # otherwise it might cause some side-effect timing issue
+    check_buffermgrd_is_running()
```

Comment:
Check the return status before stopping buffermgrd?

Reply:
Yes